crates/gosub_cairo/src/render/window.rs

```diff
@@ -0,0 +1,10 @@
+use crate::Scene;
+
+pub struct WindowData {
+    pub _cr: Option<cairo::Context>,
```

Comment:
I think this can be removed, since it is replaced by the scene?

Reply:
The scene does not know anything about the cairo context.

Due to the current setup of the renderer, we need to work with ActiveWindowData and WindowData. We do not have a generic way to send over the cr in the render function.  

Normally, you would only need to pass the cairo context (or whatever it needs to do the actual painting) at the render() step, but vello works differently.. Since there is also a `activate_window` function that creates an ActiveWindowData from WindowData, we probably need a copy of cr in the window data before we can copy it to activewindowdata..

we really should redesign the renderer so it makes more sense again for both systems.